Makefile

```
@@ -47,7 +49,7 @@ ifeq ($(USE_IMAGE_DIGESTS), true)
 endif
 
 # Image URL to use all building/pushing image targets
-IMG ?= controller:latest
+OPERATOR_IMG ?= $(IMAGE_TAG_BASE):latest
```

```suggestion
OPERATOR_IMG ?= $(IMG_BASE):$(VERSION)
```
Lets use `IMG_BASE` to stay aligned with the naming convention